### ISSUE TITLE
Print Chopped

### ISSUE THREAD

**Issue by rixtalbert** [mochizukikotaro/print-code#3]
Dear print-code
the latest release 0.6.1, 
When print-code issued to print to pdf, Text scrolls very quickly to bottom of page.
If creating pdf, only the last part is printed. The top is not printed. 
in my case lines 17 to 35 only get printed.
I do notice some short red lines "1/8 " long near the top of the print out in the left hand margin,very close to the edge of the paper.
Same thing happens when I send directly to printer.
Thanks
Rick
[print_code test.pdf](https://github.com/mochizukikotaro/print-code/files/955366/print_code.test.pdf)

**Comment by mochizukikotaro** [comment]
Hi @rixtalbert 
Thank you for your issue report. 

But... I could not reproduce the same error. 😓

https://github.com/mochizukikotaro/print-code/blob/master/lib/print-code.js#L96
```
    this.setTimeoutAsync(400)
      .then(()=>{
        atom.workspace.getActiveTextEditor().moveToTop()
        window.print()
        this.reset()
      })
```

May be,.. above code may be the cause. 

So, I will try refactor.

**Comment by mochizukikotaro** [comment]
@rixtalbert 
I released `"version": "0.6.2"`. Would you try again?
Thanks.
